src/applications/claims-status/components/ClaimsBreadcrumbs.jsx

```diff
@@ -27,7 +26,9 @@ class ClaimsBreadcrumbs extends React.Component {
 
   render() {
     return (
-      <Breadcrumbs>{this.renderBreadcrumbs(this.props.children)}</Breadcrumbs>
+      <va-breadcrumbs>
+        {this.renderBreadcrumbs(this.props.children)}
```

Comment:
NITPICK: Super minor, but this is throwing a linter error. A non-blocker, but we should be looking to reduce the linting errors.